docs/ui/prepare-migration/dataflow-config.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Configure Dataflow (Optional)
+parent: Prepare Migration Page
+grand_parent: SMT UI
+nav_order: 4
+---
+
+# Setting up connection profiles
+{: .no_toc }
+
+In case of minimal downtime migration, the dataflow jobs launched by the Spanner Migration Tool can be OPTIONALLY configured with custom runtime environment variables such as MaxWorkers, NumWorkers etc. 
+
+{: .highlight }
+The step shows configured (via a green tick) by default, denoting it is optional to configure.
+
+To configure dataflow, first specify the target database in the 'Configure Spanner Database' step. This enables the configure button for the remaining steps. The configurable fields for Dataflow are mentioned below.
+
+![](https://services.google.com/fh/files/misc/dataflow-config.png)
```

Comment:
The screenshot looks too small to understand the different fields in the form. Would suggest snipping and having only the form in the screenshot for clarity